src/main/java/frc/robot/Commands/Presets/AmpPreset.java

```diff
@@ -0,0 +1,25 @@
+package frc.robot.Commands.Presets;
```

Comment:
All these Presets commands could be part of Pivot, feels segmented to have commands that just affect pivot in a single line have a separate file.

Reply:
Possibly to rev the shooter when a preset is done or when the sensor is clicked could be put inside of the commands, which is why they are in there own file. I did not create these so I do not know the exact reason, but that is what I believe it to be. Since it works for now I don't think that this would be necessary.